### PR TITLE
create server using express-generator

### PR DESCRIPTION
**Pull request by PupS1k**

### CODE REVIEW

File: routes/login.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+var express = require('express');
+var router = express.Router();
+
+const db = require( '../public/javascripts/DateBaseUtils');
+
+router.post('/login', (req, res) => {
+  db.logInUser(req.body).then(data => res.send(data))
```
Comment (azhalkouski) [inline routes/login.js]: db.getUser(req.body.username, req.body.password).then(user => res.send(user)); ?

File: routes/products.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
+var express = require('express');
+var router = express.Router();
+const db = require( '../public/javascripts/DateBaseUtils');
+
+router.get('/kufar', (req, res) => {
```
Comment (azhalkouski) [inline routes/products.js]: '/products'?

File: app.js [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+
+app.use(function (req, res, next) {
+  res.header("Access-Control-Allow-Headers",
+      "Origin, X-Requested-With, Content-Type, Accept, Access-Control-Allow-Origin");
+  res.header("Access-Control-Allow-Origin", "*");
+  next();
+});
+
+app.use('/', indexRouter);
+app.use('/', productsRouter);
```
Comment (azhalkouski) [inline app.js]: '/products'

File: routes/products.js (hunk 2 above)
Comment (azhalkouski) [reply routes/products.js]: the whole '/products' route is handled above.
Here we can use simply '/'

File: routes/registration.js [hunk 4]
```diff
@@ -0,0 +1,10 @@
+var express = require('express');
+var router = express.Router();
+
+const db = require( '../public/javascripts/DateBaseUtils');
+
+router.post('/registration', (req, res) => {
```
Comment (azhalkouski) [inline routes/registration.js]: sign up/sign in ?

File: routes/login.js (hunk 1 above)
Comment (PupS1k) [reply routes/login.js]: fixed

File: routes/products.js (hunk 2 above)
Comment (PupS1k) [reply routes/products.js]: fixed

File: app.js (hunk 3 above)
Comment (PupS1k) [reply app.js]: fixed

File: routes/registration.js (hunk 4 above)
Comment (PupS1k) [reply routes/registration.js]: done

File: routes/login.js [hunk 5]
```diff
@@ -0,0 +1,10 @@
+var express = require('express');
+var router = express.Router();
+
+const db = require( '../public/javascripts/DateBaseUtils');
+
+router.post('/', (req, res) => {
+  db.getUser(req.body).then(data => res.send(data))
```
Comment (azhalkouski) [inline routes/login.js]: It should be GET request.

Read:
- http requests
- http request headers
- https status

Comment (azhalkouski) [reply routes/login.js]: Express error handling
https://expressjs.com/en/guide/error-handling.html

Comment (PupS1k) [reply routes/login.js]: fixed, but I do not think this option is correct

Comment (PupS1k) [reply routes/login.js]: done through headers

**Pull request merged**